routes/index.js

```diff
@@ -1,9 +1,15 @@
-var express = require('express');
+var express = require("express");
 var router = express.Router();
+const Auth = require("../utils/auth");
+const User = require("../models/User");
 
-/* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+//get dashboard
+router.get("/dashboard", Auth.verToken, (req, res) => {
+  if (req.isAdmin)
```

Comment:
Put the if statement in curly braces explicitly.

Reply:
Resolved